attachment_s3/migrations/13.0.0.0.1/post-migration.py

```diff
@@ -23,7 +23,7 @@ def migrate(cr, version):
 
     if row[0] == 's3' and bucket:
         uid = odoo.SUPERUSER_ID
-        registry = odoo.modules.registry.Registry.get(cr.dbname)
+        registry = odoo.modules.registry.RegistryManager.get(cr.dbname)
```

Comment:
I think that what you are looking for is `Registry(cr.dbname)`

Reply:
The point is that to test this slightly change, i have to deploy my swisslux instance again and again to test it out. Do you have a better way to test it btw ?